build/installEdgeDriver.ps1

```diff
@@ -0,0 +1,19 @@
+$url = "https://msedgedriver.azureedge.net/119.0.2151.72/edgedriver_win64.zip" #Chrome Driver from https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
```

Comment:
I thought using the Edge browser was simpler? Like we didn't have to install it or specify a specific version? Otherwise, if the Windows machine gets a new update with a newer Edge version, wouldn't this throw an error like with Chrome currently?

Reply:
Agreed, I just observed this. It will be maintenance hell.

Reply:
The pipeline will install both the edge driver and the required edge browser so that they are always in sync. The Edge browser is installed in OneBranch